src/molecules/formfields/CreditCardField/credit_card_field.module.scss

```diff
@@ -1,45 +0,0 @@
-$form_border: 1px solid color('neutral-5');
```

Comment:
yeah idk, I was just following the way we've been doing other formfields which is to keep all the specific styles in shared. it does seem like an anti pattern though, and that file is getting kinda big so we could extra all the specific styles into their own files & reference two sets of styles in each component?